src/main/java/codesquad/springcafe/repository/MemUserRepository.java

```diff
@@ -0,0 +1,41 @@
+package codesquad.springcafe.repository;
+
+import codesquad.springcafe.domain.User;
+import codesquad.springcafe.domain.UserDTO;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.IntStream;
+
+@Repository
+public class MemUserRepository implements UserRepository{
```

Comment:
```suggestion
public class MemoryUserRepository implements UserRepository{
```
너무 길지 않으면 줄이지 않는 편이 더 좋습니다.